packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/SelectedItemDefault.tsx

```diff
@@ -14,7 +14,8 @@ export const SelectedItemDefault: (props: IPeoplePickerItemProps) => JSX.Element
     item,
     onRemoveItem,
     index,
-    selected
+    selected,
+    removeButtonAriaLabel = ''
```

Comment:
reason to default this to empty string? do you want ariaLabel to always show up even if it is empty?